FileConv/Gfx/Fontrix-notes.md

```diff
@@ -0,0 +1,51 @@
+# Fontrix Fonts #
+
+File types:
+
+- BIN / $6400: Fontrix Font File
+
+Primary references:
+
+- Reverse engineering
```

Comment:
In other notes files I've tried to give credit to the person who did the work, e.g. https://github.com/fadden/CiderPress2/blob/main/DiskArc/FS/Gutenberg-notes.md says, "Reverse engineering, primarily by David Schmidt".  You should give yourself credit here.

Reply:
Ok. Can do! 😃